packages/neuron-ui/src/components/SUDTMigrateDialog/index.tsx

```diff
@@ -21,31 +22,56 @@ const leastSUDTAccountCapacity = BigInt(MIN_CKB_REQUIRED_BY_NORMAL_SUDT) * BigIn
 
 const SUDTMigrateDialog = ({
   cell,
+  isDialogOpen,
+  onCancel,
   openDialog,
 }: {
   cell: SpecialAssetCell
-  openDialog?: (e: React.SyntheticEvent) => void
+  isDialogOpen: boolean
+  onCancel: () => void
+  openDialog?: (type: string) => void
 }) => {
   const [t] = useTranslation()
   const isNewSUDTAccountDisabled = useMemo(() => BigInt(cell.capacity) < leastSUDTAccountCapacity, [cell.capacity])
+  const [type, setType] = useState<string>('')
+
+  const handleClick = (e: React.SyntheticEvent<HTMLDivElement>) => setType(e.currentTarget.dataset.type!)
```

Comment:
We'd better to avoid use `!`

Reply:
> We'd better to avoid use `!`

However, we will definitely receive a string here.